docs/RELEASING.md

```diff
@@ -0,0 +1,7 @@
+# Releasing
```

Comment:
Would be good to add a point with:
```
Update [CHANGELOG.md](CHANGELOG.md) with new the new release.
```

Reply:
Addressed https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/155/commits/0fda82c2e501bbac2f1f734dc1e6b0a33e60c4d1